site/archive-body.php

```diff
@@ -89,6 +87,19 @@
                             <?php echo wp_kses_post( $item->post_content ) ?>
                         <?php endforeach; ?>
                     </div>
+                    <?php
+                    $hex = p4r_get_theme_hex( $porch_fields['theme_color']['value'] );
```

Comment:
@prykon you can use PORCH_COLOR_SCHEME_HEX here instead of calculating the value again